simiyu1/JavaDevsNairobi
# 155890684 create retrofit instance

#### What does this PR do?
 - Use the retrofit Builder Class to define the base URL for network requests
#### Description of Task to be completed?
 - Create service directory
 - Create GithubService.java and define base URL

#### How should this be manually tested?
 - On navigation to the service directory and you should see the[ GithubService.java](https://github.com/simiyu1/JavaDevsNairobi/blob/ft-create-retrofit-instance-155890684/app/src/main/java/com/example/simiyunabende/javadevs/service/GithubService.java) file
 - The above file should contain the base url for requests.

#### Any background context you want to provide?
 - To send out network requests to an API, we need to use the Retrofit Builder Class and specify the base URL for the service

#### What are the relevant pivotal tracker stories?
 - [#155890684](https://www.pivotaltracker.com/story/show/155890684)
